core/Inst.hpp

```diff
@@ -32,6 +32,32 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
```

Comment:
I know I haven't set up a coding style guide, but can you ensure the style you're using matches what's already in place?  For example, the `public` keyword should align with the `class` keyword:
```suggestion
        public:
```